skore/tests/integration/ui/test_ui.py

```diff
@@ -98,3 +101,30 @@ def test_serialize_polars_series_with_missing_values(client, in_memory_project):
     assert response.status_code == 200
     project = response.json()
     assert len(project["items"]["🐻‍❄️"][0]["value"]) == 6
+
+
+def test_activity_feed(client, in_memory_project):
+    for i in range(5):
+        in_memory_project.put(f"{i}", i)
+
+    response = client.get("/api/project/activity")
+    assert response.status_code == 200
+    assert [(item["name"], item["value"]) for item in response.json()] == [
+        ("4", 4),
+        ("3", 3),
+        ("2", 2),
+        ("1", 1),
+        ("0", 0),
+    ]
+
+    now = datetime.datetime.now(tz=datetime.timezone.utc)
+
+    in_memory_project.put("4", 5)
+    in_memory_project.put("5", 5)
+
+    response = client.get("/api/project/activity", params={"after": now})
```

Comment:
Just to be sure, if I didn't add the `after` param, the test would be
```python
    assert [(item["name"], item["value"]) for item in response.json()] == [
        ("5", 5),
        ("4", 4),
        ("4", 4),
        ("3", 3),
        ("2", 2),
        ("1", 1),
        ("0", 0),
    ]
```

Reply:
but yes, you true :)

Reply:
oops yes i misread, thanks